src/WebCrawler.Shared.DevOps/CrawlerBootstrapper.cs

```diff
@@ -19,33 +21,90 @@ namespace WebCrawler.Shared.DevOps
-            return OpsConfig.GetOpsConfig().ApplyPhobosConfig().WithFallback(nextConfig);
+            // Akka.Cluster split-brain resolver configurations
+            options.SplitBrainResolver = new KeepMajorityOption();
+            
+            return options;
         }
 
-        public static Akka.Configuration.Config ApplyPhobosConfig(this Akka.Configuration.Config previousConfig)
+        public static AkkaConfigurationBuilder WithOps(this AkkaConfigurationBuilder builder, IConfiguration config)
         {
-            var enabledPhobosStr =
-                Environment.GetEnvironmentVariable(OpsConfig.PHOBOS_ENABLED)?.Trim().ToLowerInvariant() ?? "false";
-            if (bool.TryParse(enabledPhobosStr, out var enabledPhobos) && enabledPhobos)
-                return OpsConfig.GetPhobosConfig().WithFallback(previousConfig);
+            builder
+                .AddHocon(config, HoconAddMode.Prepend)
+                .BootstrapFromDocker()
+                .AddHocon(PetabridgeCmdHocon, HoconAddMode.Prepend)
+                .AddPetabridgeCmd(pbm =>
+                {
+                    // enable cluster management commands
+                    pbm.RegisterCommandPalette(ClusterCommands.Instance); 
+                })
+                // Not explicitly setting the liveness provider. The Akka.Remote port
+                // is usually an effective-enough tool for this.
+                .WithHealthCheck(opt =>
+                {
+                    // Use a second socket for TCP readiness checks from K8s
+                    opt.Readiness.Transport = HealthCheckTransport.Tcp;
+                    opt.Readiness.TcpPort = 11001;
+                });
 
-            return previousConfig;
+            return builder;
         }
 
-        /// <summary>
-        ///     Start Petabridge.Cmd
-        /// </summary>
-        /// <param name="system">The <see cref="ActorSystem" /> that will run Petabridge.Cmd</param>
-        /// <returns>The same <see cref="ActorSystem" /></returns>
-        public static ActorSystem StartPbm(this ActorSystem system)
+        public static AkkaConfigurationBuilder WithPhobos(this AkkaConfigurationBuilder builder)
         {
-            var pbm = PetabridgeCmd.Get(system);
-            pbm.RegisterCommandPalette(ClusterCommands.Instance); // enable cluster management commands
-            pbm.Start();
-            return system;
+            if (!builder.Configuration.HasValue)
+                return builder;
+
+            var config = builder.Configuration.Value;
+            if (!config.HasPath(PhobosEnabled) || !config.GetBoolean(PhobosEnabled))
+                return builder;
+            
+            builder.AddHocon(
+                ConfigurationFactory.FromResource<MarkerClass>("WebCrawler.Shared.DevOps.Config.crawler.Phobos.conf"), 
+                HoconAddMode.Prepend);
+
+
+            var statsdUrl = config.GetString(StatsdUrl);
```

Comment:
Don't worry about the StatsD stuff - it's a kind of dead technology now. Made sense in the era before OTel.

Reply:
For this sample, I'd strip out the Phobos stuff altogether.